WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using ReactiveUI;
+using System;
+using System.Collections.Generic;
+using System.Reactive;
+using System.Text;
+using WalletWasabi.Gui.ViewModels;
+
+namespace WalletWasabi.Gui.Tabs.WalletManager.HardwareWallets
+{
+	public class ConnectHardwareWalletViewModel : CategoryViewModel
+	{
+		public ConnectHardwareWalletViewModel() : base("Hardware Wallets")
+		{
+			//ConnectCommand = ReactiveCommand.CreateFromTask(LoadWalletAsync, this.WhenAnyValue(x => x.CanLoadWallet));
```

Comment:
To fix CodeFactor issue.
```suggestion
			////ConnectCommand = ReactiveCommand.CreateFromTask(LoadWalletAsync, this.WhenAnyValue(x => x.CanLoadWallet));
```

Reply:
Removed the thing.